msgboard/templates/msgboard/landing.html

```diff
@@ -0,0 +1,8 @@
+<!--msgboard/templates/msgboard/board.html-->
+
+{% extends 'base_board.html' %}
+
+{% block content %}
+You have reached landing page,
```

Comment:
this is the worst kind of a langing page, it tells you nothing about the application and asks you to "pay" with your personal details "up front"

Reply:
Ack